src/plugins/saved_objects_management/public/management_section/saved_objects_table_page.tsx

```diff
@@ -15,17 +15,22 @@ import { i18n } from '@kbn/i18n';
 import { CoreStart, ChromeBreadcrumb } from 'src/core/public';
 import { DataPublicPluginStart } from '../../../data/public';
 import { SavedObjectsTaggingApi } from '../../../saved_objects_tagging_oss/public';
+import type { SpacesAvailableStartContract } from '../../../spaces_oss/public';
 import {
   ISavedObjectsManagementServiceRegistry,
   SavedObjectsManagementActionServiceStart,
   SavedObjectsManagementColumnServiceStart,
 } from '../services';
 import { SavedObjectsTable } from './objects_table';
 
+const EmptyFunctionComponent: React.FC = ({ children }) =>
+  createElement('EmptyFunctionComponent', { children });
```

Comment:
I think `const EmptyFunctionComponent: React.FC = ({ children }) => <>{children}</>` is slightly better?